extensions/handler/message_handler.go

```diff
@@ -237,6 +247,12 @@ func (h *messageHandler) reportFirebaseLatency(latency time.Duration) {
 	}
 }
 
+func (h *messageHandler) reportRateLimitReached(game string) {
+	for _, statsReporter := range h.statsReporters {
+		statsReporter.NotificationRateLimitReached(game, "gcm")
```

Comment:
```suggestion
		statsReporter.NotificationRateLimitReached(game, "firebase")
```

Reply:
I believe it should be gcm still, as the other metrics also are